_plugins/open_external_links_in_new_tab.rb

```diff
@@ -0,0 +1,80 @@
+# This file is based on code from https://github.com/riboseinc/jekyll-external-links
```

Comment:
> Users can always open links in another tab by ctrl-click or middle mouse button. Why force how users interact with the page?

Makes sense. Changing it to external was probably a residue from my first interactions with the website. Since then I've embraced the power of ctrl-click.

Open to new tab functionality has now been removed.